README.md

```
@@ -299,6 +301,16 @@ Then, you can specify the output file with the desired extension. For example, t
 console2svg -c -d macos --timeout 5 --fps 30 --output output.mp4 -- some-command
 ```
 
+`console2svg` first checks if the detected `ffmpeg` build advertises `--enable-librsvg`. If it does, it uses `ffmpeg` directly for various conversions.
+If not, it first uses `resvg` to render SVG to PNG, and then uses `ffmpeg` to convert PNG to JPG/MP4/WebM, etc.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)